packages/cli/src/tools/windows/__tests__/androidWinHelpers.test.ts

```diff
@@ -0,0 +1,99 @@
+import * as androidWinHelpers from '../androidWinHelpers';
+import * as executeWinCommand from '../executeWinCommand';
+import * as processorType from '../../processorType';
+import {ExecaChildProcess} from 'execa';
+// Something to get the processor information
+
+describe('androidWinHelpers', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('gestBestHypervisor returns the right information depending on the system', async () => {
+    const notInstalledAndAvailable = {
+      output: `
+accel:
+0
+HAXM is not installed, but Windows Hypervisor Platform is available.
+accel`,
+      expectedResult: {
+        hypervisor: 'WHPX',
+        installed: false,
+      },
+      processor: 'Intel',
+    };
+
+    const installedAndUsable = {
+      output: `
+accel:
+0
+WHPX (10.0.19041) is installed and usable.
+accel`,
+      expectedResult: {
+        hypervisor: 'WHPX',
+        installed: true,
+      },
+      processor: 'Intel',
+    };
+
+    // TODO: Need to validate the output is correct
```

Comment:
Is this still the case?

Reply:
Yes, or at least that's what someone with AMD gave me. The whole AMD hypervisor thing is a bit "too experimental" TBH. Will remove the comment.

Reply:
Actually it is not the right output. Will update.